admin/resources/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -266,38 +276,44 @@ export const CreatePoolCandidateForm: React.FunctionComponent<CreatePoolCandidat
     );
   };
 
-export const CreatePoolCandidate: React.FunctionComponent = () => {
-  const [lookupResult] = useGetCreatePoolCandidateDataQuery();
-  const { data: lookupData, fetching, error } = lookupResult;
-  const classifications: Classification[] | [] =
-    lookupData?.classifications.filter(notEmpty) ?? [];
-  const cmoAssets: CmoAsset[] = lookupData?.cmoAssets.filter(notEmpty) ?? [];
-  const operationalRequirements: OperationalRequirement[] =
-    lookupData?.operationalRequirements.filter(notEmpty) ?? [];
-  const pools: Pool[] = lookupData?.pools.filter(notEmpty) ?? [];
-  const users: User[] = lookupData?.users.filter(notEmpty) ?? [];
+interface CreatePoolCandidateProps {
+  poolId: string;
+}
```

Comment:
Since there is only one prop for this component, we started just directly adding the type (object) directly into the type section. If you have more then 3 props then a type should be created, otherwise directly adding it is fine.

eg. `export const CreatePoolCandidate: React.FunctionComponent<{
  poolId: string;
}> = ({ poolId }) => { ... }`